README.md

```diff
@@ -168,6 +168,55 @@ flutter:
 
 These configurations will generate **`assets.gen.dart`** under the **`lib/gen/`** directory by default.
 
+#### Generate for packages
+
+If you want to generate assets for a package,
+use `package_parameter_enabled` under `flutter_gen > assets > outputs`.
+
+```yaml
+flutter_gen:
+  assets:
+    outputs:
+      package_parameter_enabled: false # <- Add this line.
```

Comment:
nit: `package_parameter_enabled` seems quite long, surely `package_parameter` would also be acceptable?

and in your example, would you set this to "true" to have the package parameter included?

Reply:
Sure. Sounds good too.